src/main.rs

```diff
@@ -107,7 +111,18 @@ pub struct CommonArgs {
 
     #[clap(short = 'v', long, action = clap::ArgAction::Count)]
     pub verbose: u8,
+
+    /// The relay URL to use as a home relay.
+    ///
+    /// If not set, sendme will use the default relay set from number 0.
+    #[clap(long)]
+    pub relay: Option<String>,
```

Comment:
Can we make this an enum?

enum Relay {
  Auto,
  Url(Url),
  Off,
}

or something?

--relay auto (default)
--relay off
--relay url(https://bla) or even just https://

?

Reply:
I've made it similar to the terminology in the code. "disabled" and "default".